test/lib/hash-state-bjj.test.js

```diff
@@ -0,0 +1,59 @@
+const path = require("path");
+const fs = require("fs");
+const Scalar = require("ffjavascript").Scalar;
+
+const tester = require("circom").tester;
+const withdrawUtils = require("@hermeznetwork/commonjs").withdrawUtils;
+
+describe("Test hash-state bjj", function () {
```

Comment:
I think this test fits better in `test/withdraw-bjj.test.js` since component `HashInputsWithdrawal` is in `src/withdraw-bjj.circom`.
We have the tests where the component is defined matching with file naming as well